index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Mock page</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div id="header">
+        <span id="menu-logo">
+            <img src="assets/images/menu-svgrepo-com.svg" class="icon" id="menu-icon">
+            <nav id="vertical-navbar">
+                <a href="" class="home-nav-link"><img src="assets/images/home-svgrepo-com.svg" alt="" class="nav-icon"><span>Home</span> </a>
+                <a href="" class="shorts-nav-link"><img src="assets/images/youtube-shorts-logo-15251.svg" alt="" class="nav-icon"><span>Shorts</span></a>
+                <a href="" class="Subscriptions-nav-link"><img src="assets/images/icons8-video-playlist-50.png" alt="" class="nav-icon"><span>Subscriptions</span></a>
+                <hr>
+                <a href="" class="Library-nav-link"><img src="assets/images/video-add-svgrepo-com.svg" alt="" class="nav-icon"><span>Library</span></a>
+                <a href="" class="History-nav-link"><img src="assets/images/history-svgrepo-com.svg" alt="" class="nav-icon"><span>History</span></a>
+                <a href="" class="videos-nav-link"><img src="assets/images/video-svgrepo-com.svg" alt="" class="nav-icon"><span>Your videos</span></a>
+                <a href="" class="watch-later-nav-link"><img src="assets/images/history-svgrepo-com.svg" alt="" class="nav-icon"><span>Watch later</span></a>
+                <div id="more" style="display: none;flex-direction: column;">
+                    <a href="" class="playlists-nav-link"><img src="" alt="" class="nav-icon">Your playlists</a>
+                    <a href="" class="some-nav-link"><img src="" alt="" class="nav-icon">Some link</a>
```

Comment:
- [ ] Kindly make sure your comments are phrased politely as suggestions instead of orders.
`This is a polite suggestion: "I think it would be better to use semantic tags"`
`Kindly consider using...`.

Reply:
I thought suggesting that the coder should "consider" the changes was quite polite. I will use your suggestions instead @christianbiring1